src/fides/api/task/create_request_tasks.py

```diff
@@ -482,8 +467,8 @@ def run_access_request(
         )
 
     for task in ready_tasks:
-        log_task_queued(task)
-        run_access_node.delay(privacy_request.id, task.id, privacy_request_proceed)
+        log_task_queued(task, "main runner")
+        queue_request_task(task, privacy_request_proceed)
```

Comment:
I created a new method, queue_request_task to figure out the particular celery task to run based on the Request Task action type so this could be shared, and also use this method to cache the celery task id in one location.